cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,55 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import { SettingsPageObject } from '../support/pages/settings.pageObject';
+
+const settingsPage = new SettingsPageObject();
 describe('Settings page', () => {
+  let user;
   before(() => {
 
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
   beforeEach(() => {
 
+    cy.task('db:clear');
+    cy.visit('/');
+    cy.login(user.email, user.username, user.password);
+    settingsPage.visit();
   });
 
   it('should provide an ability to update username', () => {
 
+    settingsPage.typeUsername(user.username);
+    settingsPage.submitForm();
+    settingsPage.assertProfilePage(user.username);
   });
 
   it('should provide an ability to update bio', () => {
 
+    settingsPage.typeBio(user.bio);
+    settingsPage.submitForm();
```

Comment:
Consider adding an assertion here to verify that the bio has been updated successfully on the profile page.